### DISCUSSION TITLE
Add extra space to the end of a transient prompt

### DISCUSSION THREAD

**Discussion by bleuthoot-sven** [JanDeDobbeleer/oh-my-posh#2120] [Troubleshoot] [answered]
Hello,

I have been trying to add a new line to the end of a transient prompt, so the commands will be a bit more spaced out. I kind of want it to look like how it is done with Powerlevel10k

How it currently look:
![image](https://user-images.githubusercontent.com/25549100/164448909-9b13f6c0-8bdc-4479-be99-1b168acafe42.png)

How I want it to look:
![image](https://user-images.githubusercontent.com/25549100/164448802-d787a2cb-f9c8-439a-850c-f1453d549de1.png)

How can I achieve this?

The closest I got was adding this block to the top, but this will also add a new line when I open PowerShell:

```json
{
    "type": "prompt",
    "alignment": "left",
    "segments": [
        {
            "type": "text",
            "style": "plain",
            "template": "\u0000"
        }
    ],
    "newline": true
}
```

How it looks:

![image](https://user-images.githubusercontent.com/25549100/164451175-1547f6a1-61e3-49c3-9913-54a148334f80.png)
![image](https://user-images.githubusercontent.com/25549100/164451252-e8c5f596-ca9c-4555-be95-1dbe133559b6.png)

**Comment by JanDeDobbeleer** [comment, marked as answer]
The only way to reach this is to add a newline to the first block indeed. This is a reload prompt, so we have no influence of what's added after the command.

**Reply by bleuthoot-sven** [reply]
So for my understanding:

What I have is the closest I can to adding a new line after every command. I cannot remove that first new line when I start up powershell?

Is that correct?

**Reply by JanDeDobbeleer** [reply]
Correct. As we only print the prompt part, we don't influence the command output itself.

**Reply by bleuthoot-sven** [reply]
> Correct. As we only print the prompt part, we don't influence the command output itself.

Thanks for the info. Guess I have to live with this minor inconvenience :)

**Comment by Rishav-sy** [comment]
This worked for me, 
add this to the start of block :- 

  "blocks": [
    {
      "type": "prompt",
      "alignment": "left",
      "segments": [
        {
          "template": "\n➜
the arrow was already there for my theme maybe, i just added \n

from
     
<img width="508" height="145" alt="Control-V(2)" src="https://github.com/user-attachments/assets/b841c482-8600-4f2b-bfa2-afe6c8553d99" />
  
 to 
  
  
<img width="530" height="140" alt="Control-V(1)" src="https://github.com/user-attachments/assets/4c92e7e1-9e42-4e65-91b6-92209f7b1515" />
